about.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="stylesheets/styles.css">
+  <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css"
+  href="https://fonts.googleapis.com/css?family=Lobster|Harmattan|Josefin+Sans|Abel|Lato|Playfair+Display">
+  <title>About Me</title>
+</head>
+
+<body id="page3">
+  <header>
+    <div class="header-name">
+      <a href="about.html" target="_blank">Haby Randall</a>
+    </div>
+    <nav class="top-nav">
+      <ul>
+        <li class="btn"> <a href="hobby-blog.html" target="_blank">Hobbies</a></li>
+        <li class="btn"> <a href="porfolio.html" target="_blank">Porfolio</a></li>
+        <li class="btn"> <a href="about.html">About Me</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section>
+      <article class="bio">
+        <img src="images/IMG_4566.JPG" />
```

Comment:
adding a description under an "alt" attribute to image tags can be helpful in case links to images are broken.